### DISCUSSION TITLE
Does Renovate support scanning indirect dependencies of Java projects?

### DISCUSSION THREAD

**Discussion by linzh92** [renovatebot/renovate#23610] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.117

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Extracting indirect dependencies of Java project (a dependency of dependencies).

### What you tried so far.

Searching and reading relevant information about renovate.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
No, Renovate only cares about dependencies it can update

**Reply by linzh92** [reply]
Thx a lot.
